source/changelog.txt

```diff
@@ -24,6 +24,13 @@ v2.0.0
   - ``db.enableFreeMonitoring``
   - ``db.disableFreeMonitoring``
 
+v1.10.5
+-------
+*Released August 11, 2023*
+
+Provides an accessible list of ``mongosh`` `downloads <https://downloads.mongodb.com/compass/mongosh.json>`__ 
```

Comment:
```suggestion
Provides a programmatically accessible list of ``mongosh`` `downloads <https://downloads.mongodb.com/compass/mongosh.json>`__ 
```

I think a lot of people might assume that “accessible” applied to a URL refers to accessibility in the context of web design, not programmatic access (which is kind of the opposite)